src/Core/Localization/CLDR/Locale.php

```diff
@@ -212,7 +212,7 @@ public function getDecimalPattern($numberingSystem = null)
             throw new LocalizationException('No decimal pattern found for numbering system: ' . $numberingSystem);
         }
 
-        return $this->decimalPatterns[$numberingSystem];
+        return $this->decimalPatterns[$numberingSystem] ?? '';
```

Comment:
```suggestion
        return $this->decimalPatterns[$numberingSystem];
```

Reply:
The previous isset make the job

Reply:
I'm sure I needed to do this when I started working on this PR, but I agree that it doesn't make sense and it runs well without my changes. gonna revert that!